openmetadata-ui/src/main/resources/ui/src/components/Pipeline/Execution/execution.less

```diff
@@ -11,21 +11,31 @@
  *  limitations under the License.
  */
 
-.executions-date-picker {
+.ant-picker-range.executions-date-picker {
+  padding: 0 6px 0 0;
+  max-width: 200px;
   .ant-picker-range-separator {
     display: none;
   }
   .ant-picker-clear {
-    right: 4px;
+    right: -6px;
+  }
+
+  .ant-picker-active-bar {
+    margin-left: 0;
+    width: 85px !important;
```

Comment:
why we are giving !important?

Reply:
so, the active element is having inline css, which is only overrideable using important!